src/pages/user/institution/InstitutionCard.tsx

```diff
@@ -59,6 +71,15 @@ const InstitutionCard: FC<InstitutionCardProps> = ({ unit }) => {
     }
   };
 
+  const handleEditInstitution = async () => {
+    onEdit();
```

Comment:
rename til `closeEdit()` elns? `onEdit` kan være hva som helst og man må åpne annen fil for å se hva som ligger der

Reply:
Jeg ser poenget ditt, men det er vanlig praksis å kalle action props for `onAction` også der det brukes så kaller man det `handleAction`. Det som skjer her er jo at EditMode aktiveres, så det er jo det motsatte av closeEdit egentlig.

Reply:
ja, my bad
men ok, daaa :)